storage/innobase/xtrabackup/test/t/xbcloud.sh

```diff
@@ -113,23 +117,23 @@ if ! grep -q failed $topdir/pxb-1832.log ; then
 fi
 
 #PXB-2164 xbcloud doesn't return the error if the backup doesn't exist in s3 bucket
-xbcloud --defaults-file=$topdir/xbcloud.cnf get somedummyjunkbackup 2>$topdir/pxb-2164.log
+run_cmd_expect_failure xbcloud --defaults-file=$topdir/xbcloud.cnf get somedummyjunkbackup 2>$topdir/pxb-2164.log
```

Comment:
has this changed on this PR?
The comment say's xbcloud does not return error, and we are executing `run_cmd_expect_failure`

Reply:
The comment is about jira ticket so may be misleading,  It should fail and return error. The problem is seen only with macos when xbcloud fails the testcase stops. so added run_cmd_expect

Reply:
Can you please make sure it works on other platforms too?
From `xbcloud` to `run_cmd_expect_failure` we are changing from expecting success vs error. I remember centos7 been more sensitive for those things.

Reply:
I think previously the expected error wasn't captured.